packages/sdk/phone-utils/src/io.ts

```diff
@@ -14,23 +14,6 @@ export const E164PhoneNumberType = new t.Type<string, string, unknown>(
     ),
   String
 )
-export const AttestationServiceStatusResponseType = t.type({
```

Comment:
moved to Utils, so that AttewstationsWrapper does not need to import phone-utils